src/view/atom.ts

```diff
@@ -35,7 +35,7 @@ export function confirmPackagesToInstall({
           text: 'Yes',
           onDidClick: () => {
             if (skipGroups) {
```

Comment:
Side note, this whole section, i.e.
```js
if (skipGroups) {...} else { ... }
```
is behaviourally equivalent to just the `else` block, since when `skipGroups === true`, `groupChoices === []`. But the variant with the branching may be slightly more efficient since it doesn't copy the array.